pom.xml

```diff
@@ -1541,6 +1542,19 @@
               <fail>true</fail>
             </configuration>
           </execution>
+          <execution>
+            <id>enforce-java</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <rules>
+                <requireJavaVersion>
+                  <version>[1.8.0,1.9.0)</version>
```

Comment:
Is this the right syntax? has an opening square bracket but closes with a parenthesis? Dont we want to limit to a single minimum version?
Also shouldn't the same fix be applied to say standalone-metastore/pom.xml and itests/pom.xml as well ?

Reply:
This sets the range to 1.8.0 <= x < 1.9.0 -- https://maven.apache.org/enforcer/enforcer-rules/versionRanges.html

I can apply the change to the other files...

Reply:
I'm going to vote against applying the same changes to those files, at present they don't appear to include `maven-enforcer-plugin`. If someone wants to include that, they're welcome to copy the entire plugin from this file after it merges.